### ISSUE TITLE
[Issue]: FSP Fails to Load Any Game with v0.6r1476

### ISSUE THREAD

**Issue by j117-esc** [emukidid/swiss-gc#810]
### Describe the bug

After upgrading to v0.6r1476, FSP fails to load any game and an error message is shown:
'An error has occurred. Turn the power OFF and check the NINTENDO GAMECUBE Instruction Booklet for further instructions.'

FSP works again after reverting to previous release v0.6r1458.

Currently using the official GCLoader (2.0.0) to boot into Swiss.

### Steps to reproduce

Upgrade to v0.6r1476, load FSP through broadband adapter, select any game to begin streaming and error message shows up.

### Expected behavior

Game should load without issue as was the case in the prior release v0.6r1458.

### Relevant log output

```shell
'An error has occurred. Turn the power OFF and check the NINTENDO GAMECUBE Instruction Booklet for further instructions.'
```

**Comment by Extrems** [comment]
Cannot reproduce.

**Comment by Extrems** [comment]
In my case all stuttering in all known titles is gone. What kind of hardware are you using?

**Comment by j117-esc** [comment]
I'm currently using a RockPro64 to hold all my games and stream ps3/gamecube games.
After testing out more games, the CGI stutters I used to experience overall has improved significantly. I'm only seeing Sunshine having a few seconds of stuttering in the intro where I wasn't before, but every other game I have tested has either minimized stuttering or no longer stutters on intros.

My apologies for not doing more research beforehand, things are definitely running smoother now with this latest release.
